canarieapi/api.py

```diff
@@ -62,7 +62,7 @@
         get_db()
 
 CronAccessStats = TypedDict("CronAccessStats", {
-    "invocations": int,
+    "invocations": Optional[int],  # count | Not monitored
```

Comment:
`CronAccessStats` always returns an `int` for `invocations`. It is simply ignored and not set into the monitoring info if `PARSE_LOGS=True`.